src/main/resources/public/js/components/dropdown_menus/member/member-dropdown-header.js

```diff
@@ -96,6 +96,16 @@ class MemberDropdownHeader extends React.Component {
                         </CustomTooltip>
                     }
 
+                    {
```

Comment:
this should not be part of the pull request (which must only deal with the "create organization" button)